### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by olenitut**
- [DEMO LINK](https://olenitut.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://olenitut.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```
Comment (stanislav-topikha) [inline src/style.css:4]: fallback fonts

File: src/style.css [hunk 2]
```diff
@@ -1 +1,64 @@
...
+  box-sizing: border-box;
```
Comment (stanislav-topikha) [inline src/style.css:5]: set it in * selector

File: src/style.css [hunk 3]
```diff
@@ -1 +1,64 @@
...
+}
+
+.page-header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  /* align-items: center; */
+}
```
Comment (stanislav-topikha) [inline src/style.css:13]:
```suggestion
  justify-content: space-between;
}
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+.nav-list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  /* gap: 20px; */
+  list-style: none;
```
Comment (stanislav-topikha) [inline src/style.css:20]:
```suggestion
.nav-list {
  margin: 0;
  padding: 0;
  display: flex;
  list-style: none;
```

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
Visited links should be black
image

this one should not be blue always
image

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
visited links should be black 
image

File: src/style.css [hunk 5]
```diff
@@ -1 +1,65 @@
 /* Styles go here */
+* {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  box-sizing: border-box;
+}
```
Comment (stanislav-topikha) [inline src/style.css:6]:
```suggestion
* {
   box-sizing: border-box;
}

html {
  font-family: "Roboto", Helvetica, sans-serif;
}

body {
  marging: 0;
}
```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,65 @@
...
+
+.nav-link:visited {
+  color: #000;
+}
+
+.nav-link:hover,
+.nav-link:active,
+.is-active:link {
+  color: #00acdc;
```
Comment (stanislav-topikha) [inline src/style.css:47]: lets add var for repetitive color value

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 5 above)
Comment (SanyaBratashchuk) [reply src/style.css:11]: Fix it

File: src/style.css [hunk 7]
```diff
@@ -1 +1,70 @@
...
+  list-style: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav-link:link {
+  position: relative;
+  display: flex;
+  height: 60px;
```
Comment (SanyaBratashchuk) [inline src/style.css:32]: do not hardcode height, use vertical paddings

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by AlisaVlasova — APPROVED** [review]

**Pull request closed without merging**